packages/devtools_app/lib/src/memory/flutter/memory_filter.dart

```diff
@@ -0,0 +1,391 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:vm_service/vm_service.dart';
+
+import '../../flutter/auto_dispose_mixin.dart';
+import '../../flutter/flutter_widgets/linked_scroll_controller.dart';
+
+import 'memory_controller.dart';
+import 'memory_snapshot_models.dart';
+
+/// First two libraries are special e.g., dart:* and package:flutter*
+const String _dartLibraryUriPrefix = 'dart:';
```

Comment:
these types can be removed

Reply:
Why do you think these can be removed they are special for wildcard filtering.